tests/features/bootstrap/FeatureContext.php

```diff
@@ -29,6 +35,229 @@ public function assertAccessDenied() {
   }
 
   /**
+   * Assert that certain fields are present on the page.
+   *
+   * @param string $fields
+   *    Fields.
+   *
+   * @throws \Exception
+   *   Thrown when an expected field is not present.
+   *
+   * @Then /^(?:|the following )fields should be present? "(?P<fields>[^"]*)"$/
```

Comment:
I was actually wondering why too. But this is code moved around so I didn't focus on it.